HACKING.md

```diff
@@ -5,6 +5,10 @@ Here's where to get the code:
     git clone https://github.com/cockpit-project/cockpit
     cd cockpit/
 
+Cockpit uses a lot of build and runtime dependencies from [npm](https://www.npmjs.com). Install them with
```

Comment:
It's not really 'Install them', it's more 'Download them'.

Reply:
Good point -- Changed to "Get them".